js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
+  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+
+  defaultStyle();
+
+  if (this.value === '') {
+    styleSeting('email', 'empty');
+  } else if (!emailRegEx.test(this.value)) {
+    styleSeting('email', 'wrong');
+  }
```

Comment:
조건식에서 else-if 보다는 if로 이렇게 작성해주는게 더 좋을 것 같아요!

```suggestion
  if (this.value === '') {
    styleSeting('email', 'empty');
  }
  
  if (!emailRegEx.test(this.value)) {
    styleSeting('email', 'wrong');
  }
```